### PR TITLE
Added a Weather App

### PR DESCRIPTION
**Pull request by shruti-sen2004**
Title :Added a Weather App

Code Stack : HTML, CSS and javascript

# Checklist:

- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have created a helpful and easy to understand `README.md`
- [X] I have updated the Index.html file for my contribution

# Demo image:  

image

# Website Link: 
To test my website click here: https://shruti-sen2004.github.io/WeatherApp/

### CODE REVIEW

File: native/machine-coding/Weather App/style.css [hunk 1]
```diff
@@ -0,0 +1,111 @@
+@import url(../../helpers/core.css);
+
+*{
+    margin: 0;
+    padding: 0;
+    font-family: 'Poppins', sans-serif;
+    box-sizing: border-box;
+}
```
Comment (sadanandpai) [inline native/machine-coding/Weather App/style.css:8]: above css ruleset is not needed as we import core

File: native/machine-coding/Weather App/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    }
+    else if(data.weather[0].main == "Rain"){
+        weatherIcon.src = "images/rain.png"
+    }
+    else if(data.weather[0].main == "Drizzle"){
+        weatherIcon.src = "images/drizzle.png"
+    }
+    else if(data.weather[0].main == "Mist"){
+        weatherIcon.src = "images/mist.png"
+    }
```
Comment (sadanandpai) [inline native/machine-coding/Weather App/index.js:39]: can we not directly use the value of man here

weatherIcon.src = `images/${data.weather[0].main.toLowerCase()}.png`

File: native/machine-coding/Weather App/index.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        document.querySelector(".weather").style.display = "none";
+    }
+
+    else{
+    var data = await response.json();
+   
+    document.querySelector(".city").innerHTML = data.name;
+    document.querySelector(".temp").innerHTML = Math.round(data.main.temp) + "°C";
+    document.querySelector(".humidity").innerHTML = data.main.humidity + "%";
+    document.querySelector(".wind").innerHTML = data.wind.speed + "km/h";
```
Comment (sadanandpai) [inline native/machine-coding/Weather App/index.js:23]: does it work if we keep the selectors outside the function?

File: native/machine-coding/Weather App/index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                    <p>Wind Speed</p>
+                </div>
+            </div>
+        </div>
+    </div>
+</div>
+<script src="index.js"></script>
+</body>
+</body>
+</html>
```
Comment (sadanandpai) [inline native/machine-coding/Weather App/index.html:46]: please format and indent

**Review by sadanandpai — APPROVED** [review]

**Review by sadanandpai — CHANGES_REQUESTED** [review]
Please do the changes as per the comments

**Comment by shruti-sen2004** [comment]
Okay

File: native/machine-coding/Weather App/style.css (hunk 1 above)
Comment (shruti-sen2004) [reply native/machine-coding/Weather App/style.css:8]: okay

File: native/machine-coding/Weather App/index.js (hunk 2 above)
Comment (shruti-sen2004) [reply native/machine-coding/Weather App/index.js:39]: So i tried doing this but it was as it was not properly functional thus i didnot made this change

File: native/machine-coding/Weather App/index.js (hunk 3 above)
Comment (shruti-sen2004) [reply native/machine-coding/Weather App/index.js:23]: no its not properly functional like that..........

File: native/machine-coding/Weather App/index.html (hunk 4 above)
Comment (shruti-sen2004) [reply native/machine-coding/Weather App/index.html:46]: okay

**Comment by shruti-sen2004** [comment]
I have made the changes that were requested

**Review by sadanandpai — APPROVED** [review]

**Pull request merged**